README.md

```diff
@@ -10,10 +10,21 @@ Downloads the following materials:
 * Pages and Labels: Embedded Opencast and Youtube Videos
 
 # Setup
-Initially you need to install the requirements (**Python version >=3.6 is needed.**):
+This software requires **Python version >=3.6**.
+
+Obtain the source, for example using `git clone` or downloading and unzipping a zip. The following exemplary commands are to be used from inside that downloaded `syncMyMoodle` directory.
+
+Since you need to install dependencies to run syncMyMoodle, the recommended way to install them is by first creating and activating a virtual environment ([more info](https://packaging.python.org/guides/installing-using-pip-and-virtual-environments/#creating-a-virtual-environment)):
+```bash
+python3 -m venv moodle-venv
+source moodle-venv/bin/activate
+```
+
+Then install the requirements:
 ```bash
 pip3 install -r requirements.txt
 ```
+It is recommended to also install and use the [FreeDesktop.org Secret Service integration](#freedesktoporg-secret-service-integration) if your system supports it.
```

Comment:
Citation needed ;P

Reply:
I don't believe there is an argument that storing the pw plain is better than in the encrypted keyring your system already offers?
Also passing pw on cli *is* storing it plain... in your history, unless you take special action around it.

Reply:
Yes I am definitely with you however it adds additional steps to the setup process which are not required for a quick start. That being said I definitely agree that we should advocate the usage of a keyring albeit only after all the mandatory steps.

Reply:
- I mentioned it there because it is right after the installation of the other deps, and I feel like it's less likely a user will go back to install more stuff
- other than that the rest of SSI setup means only activating it in the conf
- idk the SSI dep could also be in the default `requirements.txt` (not bother users with it) and then you need only enable it. doesn't really hurt, but it's less clean. we could have a `requirements_optional.txt`?